### DISCUSSION TITLE
Cobe

### DISCUSSION THREAD

**Discussion by cobe1494** [Cog-Creators/Red-DiscordBot#5072] [General] [answered]
var rpc = require("discord-rpc")
const { ClientId, Button1Label, Button1Link, LargeImage, LargeImageText, State, Details } = require("./config.json")
const client = new rpc.Client({ transport: 'ipc' })
client.on('ready', () => {
client.request('SET_ACTIVITY', {
pid: process.pid,
activity : {
details : Details,
state : State,
assets : {
large_image : LargeImage,
large_text : LargeImageText
},
buttons : [{label : Button1Label , url : Button1Link}]
}
})
})
client.login({ clientId : ClientId }).catch(console.error);

**Comment by Jackenmen** [comment, marked as answer]
This is not a place to ask support questions related to what I presume is discord.js.
